### DISCUSSION TITLE
Symfony UX Twig Components

### DISCUSSION THREAD

**Discussion by plixus** [pimcore/pimcore#17155] [Q&A / Support] [answered]
Is anyone of you already using [Symfony UX Twig Components](https://symfony.com/bundles/ux-twig-component/current/index.html) in a Pimcore project? I have the problem that the classes of the components are not found. 

See: [https://github.com/symfony/ux/issues/1895](https://github.com/symfony/ux/issues/1895)

**Comment by dpfaffenbauer** [comment]
we do, maybe @breakone can help here?

**Comment by breakone** [comment, marked as answer]
@plixus 
do you register the components in your service.yaml?

```
services:
...
    App\Twig\Components\:
        resource: '../../src/App/Twig/Components'
...
```

**Reply by plixus** [reply]
Oh, ha, that's it! 
Thank you!
